.github/workflows/bot-comment-notify.yml

```
@@ -0,0 +1,148 @@
+  notify-slack:
+    runs-on: ubuntu-latest
+    # PRまたはIssueへのコメントで、CodeRabbitまたはGitHub Actionsボットからのコメントのみを対象とする
+    if: |
+      (github.event.issue.pull_request || github.event.issue) &&
+      (contains(github.event.comment.user.login, 'coderabbit') ||
+       github.event.comment.user.login == 'github-actions[bot]')
+
+    steps:
+      - name: Determine bot type
+        id: bot-type
+        run: |
```

actionlint (1.7.7)

"github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)